### PR TITLE
polybar: 3.5.7 -> 3.6.1

### PR DESCRIPTION
**Pull request by moni-dz**
###### Motivation for this change
changelog: https://github.com/polybar/polybar/releases/tag/3.6.0
see also: https://github.com/polybar/polybar/issues/1971#issuecomment-1055754773

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fortuneteller2k** [comment]
Result of `nixpkgs-review pr 162550` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>polybar</li>
    <li>polybarFull</li>
  </ul>
</details>

**Comment by fortuneteller2k** [comment]
Result of `nixpkgs-review pr 162550` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>polybar</li>
    <li>polybarFull</li>
  </ul>
</details>

**Review by jfly — APPROVED** [review]
LGTM! :shipit:

**Comment by fortuneteller2k** [comment]
Result of `nixpkgs-review pr 162550` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>polybar</li>
    <li>polybarFull</li>
  </ul>
</details>

**Comment by jtrees** [comment]
This release introduces several breaking changes, right? Should it be added to the changelog?

**Comment by Lassulus** [comment]
> This release introduces several breaking changes, right? Should it be added to the changelog?
Would be nice. Can you do that?

File: nixos/doc/manual/release-notes/rl-2205.section.md [hunk 1]
```diff
@@ -364,6 +364,8 @@ In addition to numerous new and upgraded packages, this release has the followin
   - The RPC protocol version was bumped; all zrepl daemons in a setup must be updated and restarted before replication can resume.
   - A bug involving encrypt-on-receive has been fixed. Read the [zrepl documentation](https://zrepl.github.io/configuration/sendrecvoptions.html#job-recv-options-placeholder) and check the output of `zfs get -r encryption,zrepl:placeholder PATH_TO_ROOTFS` on the receiver.
 
+- The `polybar` package has been updated from 3.5.7 to 3.6.2. See [the changelog](https://github.com/polybar/polybar/releases/tag/3.6.0) for more details.
```
Comment (Lassulus) [inline nixos/doc/manual/release-notes/rl-2205.section.md:367]: maybe we should mention something about breaking changes

Comment (aforemny) [inline nixos/doc/manual/release-notes/rl-2205.section.md:367]:
```suggestion
- The `polybar` package has been updated from 3.5.7 to 3.6.2. See [the changelog](https://github.com/polybar/polybar/releases/tag/3.6.0) for more details.
  - Breaking changes include changes to escaping rules in configuration values, changes in behavior when encountering invalid tag names, and changes to inter-process-messaging (IPC).
```

**Review by aforemny — APPROVED** [review]

**Review by aforemny — CHANGES_REQUESTED** [review]
It seems that now a lot of noise is being generated in the generated NixOS manual. Do we know the offending commit?

I would rather not include that noise into this commit.

**Comment by aforemny** [comment]
IMO we should rebase this after https://github.com/NixOS/nixpkgs/pull/168596 is merged, and we are good to go here!

**Comment by aforemny** [comment]
@fortuneteller2k Could you rebase and re-generate docs again now that #168598 is merged?

**Pull request merged**